Watchman.Web/Areas/JwtAuth/Infrastructure/RefreshToken.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace Watchman.Web.Areas.JwtAuth.Infrastructure
```

Comment:
raczej Infrastructure.JwtAuth

Reply:
Planowałem w JwtAuth później też zrobić logowanie, ale to słabe rozwiązanie. Zrobię tak jak napisałeś, w razie czegoś będzie, można w tym Infrastructure dodać weryfikacje maila itp. A login zrobić trzeba osobno.